### PR TITLE
modify the error url of volumesnapshot

### PR DESCRIPTION
**Pull request by timyinshi**
/kind bug
/sig docs

the error url is：
https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go
https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/create_crd.go

the right url is:
https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1beta1/types.go
https://github.com/kubernetes-csi/external-snapshotter/blob/release-1.2/cmd/csi-snapshotter/create_crd.go

### CODE REVIEW

File: book/src/snapshot-restore-feature.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Similar to the API for managing [Kubernetes Persistent Volumes](https://kubernet
...
 
-The CRDs are [automatically deployed](https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/create_crd.go#L29) by the CSI [external-snapshotter](external-snapshotter.md) sidecar.
+The CRDs are [automatically deployed](https://github.com/kubernetes-csi/external-snapshotter/blob/release-1.2/cmd/csi-snapshotter/create_crd.go#L29) by the CSI [external-snapshotter](external-snapshotter.md) sidecar.
 
-The schema definition for the custom resources (CRs) can be found here: [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go)
+The schema definition for the custom resources (CRs) can be found here: [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1beta1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go)
```
Comment (msau42) [inline book/src/snapshot-restore-feature.md]: This should refer to the alpha crds, not the beta ones.

/assign @xing-yang @yuxiangqian
this whole section will need to be redone for beta. Do we want to do that now?

Comment (xing-yang) [reply book/src/snapshot-restore-feature.md]: We need to re-do this whole section after we cut the 2.0 release.

Comment (timyinshi) [reply book/src/snapshot-restore-feature.md]: @xing-yang ok,thanks

**Pull request closed without merging**